desub-current/src/decoder/mod.rs

```diff
@@ -477,44 +503,76 @@ pub struct ExtrinsicSignature<'a> {
 	/// Signed extensions, which can vary by node. Here, we
 	/// return the name and value of each.
 	#[serde(borrow)]
-	pub extensions: Vec<(Cow<'a, str>, Value)>,
+	pub extensions: Vec<(Cow<'a, str>, Value<T>)>,
 }
 
-impl<'a> ExtrinsicSignature<'a> {
-	pub fn into_owned(self) -> ExtrinsicSignature<'static> {
+impl<'a, T> ExtrinsicSignature<'a, T> {
+	pub fn into_owned(self) -> ExtrinsicSignature<'static, T> {
 		ExtrinsicSignature {
 			address: self.address,
 			signature: self.signature,
 			extensions: self.extensions.into_iter().map(|(k, v)| (Cow::Owned(k.into_owned()), v)).collect(),
 		}
 	}
+	pub fn without_context(self) -> ExtrinsicSignature<'a, ()> {
+		ExtrinsicSignature {
+			address: self.address,
+			signature: self.signature,
+			extensions: self.extensions.into_iter().map(|(k,v)| (k, v.without_context())).collect()
+		}
+	}
```

Comment:
When you have containers over arbitrary `T`s, they are usually a Functor. You can write a `map` function for this type which applies an arbitrary function over all the `T`s.

Reply:
I could, although I have removed the parameterization as of a more recent commit and so this is a moot point right here (though worth a thought for the `Value<T>` type!)  :)

Reply:
I added `map_context` to the `Value` types; it seemed like a reasonable thing to want if you need to map the context to a different type!